src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/metrikker/BedriftsmetrikkerController.java

```diff
@@ -33,14 +38,19 @@ public Bedriftsmetrikker hentBedriftsmetrikker(
             HttpServletRequest request
     ) {
 
-        Orgnr orgnr = new Orgnr(orgnrStr);
-        Underenhet underenhet = enhetsregisteretClient.hentInformasjonOmUnderenhet(orgnr);
-        Optional<Bransje> bransje = bransjeprogram.finnBransje(underenhet);
+        try {
+            Orgnr orgnr = new Orgnr(orgnrStr);
+            Underenhet underenhet = enhetsregisteretClient.hentInformasjonOmUnderenhet(orgnr);
+            Optional<Bransje> bransje = bransjeprogram.finnBransje(underenhet);
 
-        return Bedriftsmetrikker.builder()
-                .antallAnsatte(new BigDecimal(underenhet.getAntallAnsatte()))
-                .næringskode5Siffer(underenhet.getNæringskode())
-                .bransje(bransje.isPresent() ? bransje.get().getType() : null)
-                .build();
+            return Bedriftsmetrikker.builder()
+                    .antallAnsatte(new BigDecimal(underenhet.getAntallAnsatte()))
+                    .næringskode5Siffer(underenhet.getNæringskode())
+                    .bransje(bransje.isPresent() ? bransje.get().getType() : null)
+                    .build();
+        } catch (Exception e) {
+            logger.warn("Feil ved kall til Enhetsregisteret");
```

Comment:
Dette blir litt feil - nå får brukeren uansett null hvis det er noe feil i dette endepunktet. Vi vil f.eks. at de skal få 403 hvis man ikke har tilgang. Og hvis det faktisk blir en feil, så burde man få 500 i stedet for null. Vi burde kanskje tenke litt om igjen på dette 🤔